src/components/combobox/combobox.tsx

```diff
@@ -419,14 +423,16 @@ export class Combobox implements LabelableComponent, FormComponent, InteractiveC
         }
         break;
       case "Home":
-        event.preventDefault();
-        this.active = true;
+        this.active ? event.preventDefault() : null;
         this.updateActiveItemIndex(0);
+        this.activeInView();
+        !inViewport ? this.el.scrollIntoView({ block: "start", inline: "nearest" }) : null;
```

Comment:
Unfortunately, `scrollIntoView` in Safari does not support the options param: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#browser_compatibility.